onboarding/source/amb_light_service.c

```diff
@@ -18,22 +25,57 @@ static void lightServiceTask(void * pvParameters);
 uint8_t initLightService(void) {
     /* USER CODE BEGIN */
     // Create the task and queue here.
+    BaseType_t xReturned = pdFAIL;
+
+    // Create light task
+    if (lightServiceHandle == NULL) {
+        xReturned = xTaskCreate(lightServiceTask,
+                                LIGHT_SERVICE_NAME,
+                                LIGHT_SERVICE_STACK_SIZE,
+                                NULL,
+                                LIGHT_SERVICE_PRIORITY,
+                                &lightServiceHandle);
+    }
+    
+    if (xReturned == pdFAIL) {
+            printf("\nlightServiceTask not created...\n");
+    }
+
+    // Create queue
+    if (lightServiceQueue == NULL) {
+        lightServiceQueue = xQueueCreate(LIGHT_QUEUE_LENGTH, LIGHT_QUEUE_LENGTH);
+    }
```

Comment:
Check if the queue was created successfully. It can fail if we've run out of memory.